mcs/class/corlib/ReferenceSources/RuntimeType.cs

```diff
@@ -544,14 +550,14 @@ public override InterfaceMapping GetInterfaceMap (Type ifaceType)
 
 			InterfaceMapping res;
 			if (!ifaceType.IsInterface)
-				throw new ArgumentException (Locale.GetText ("Argument must be an interface."), "ifaceType");
+				throw new ArgumentException ("Argument must be an interface.", "ifaceType");
```

Comment:
I don't understand why it's ok to remove these calls to `Locale.GetText` - isn't this changing something for our corlib?